Added quotes (#372)
deltachat/deltachat-pages: en/user-voices.md

**Blind user** about the Android version in 2019, via e-mail to delta@merlinux.eu

### Popular in regions with strict Internet governance

> _"There's this nice messenger, based on a simple email, called DeltaChat. I am happy that our Internet watchdog has made this app popular by trying to block it. It's just like with Telegram that has become widely popular across non-tech audiences after the ban"._ 

Reviewer: ```suggestion
_"There's this nice messenger, based on a simple email, called Delta Chat. I am happy that our Internet watchdog has made this app popular by trying to block it. It's just like with Telegram that has become widely popular across non-tech audiences after the ban"._ 
```

> **Russian user** from 2020 in response to Delta.Chat's rejection to satisfy Russian Internet watchdog RKN's request for user data. (Twitter, https://twitter.com/Alex0s/status/1256841124427313153)

Reviewer: ```suggestion
**Russian user** from 2020 in response to Delta Chat's rejection to satisfy Russian Internet watchdog RKN's request for user data, via [Twitter](https://twitter.com/Alex0s/status/1256841124427313153)
```

### Chatty interface with robust encryption

_"Other things I know I'm late to the party on but delta-chat seems really neat, especially if you have privacy concerns, since it's really just a specialized encryption heavy email client that presents a chat interface"_

> **American user** from 2020 recommending Delta.Chat for everyday usage

Reviewer: ```suggestion
**American user** from 2020 recommending Delta Chat for everyday usage
```
